ga4gh/datamodel/variants.py

```diff
@@ -450,9 +450,10 @@ def convertVariant(self, record):
         if record.id is not None:
             variant.names = record.id.split(';')
         variant.start = record.start          # 0-based inclusive
-        variant.end = record.stop             # 0-based exclusive
+        variant.end = record.start + len(record.ref)
```

Comment:
The GA4GH docs are technically correct, but the qualifier "typically" hides
important details.  :)

On Fri, Mar 6, 2015 at 12:52 PM, Jerome Kelleher <notifications@github.com>
wrote:

> In ga4gh/datamodel/variants.py
> <https://github.com/ga4gh/server/pull/211#discussion_r25962648>:
>
> > @@ -450,9 +450,10 @@ def convertVariant(self, record):
> >          if record.id is not None:
> >              variant.names = record.id.split(';')
> >          variant.start = record.start          # 0-based inclusive
> > -        variant.end = record.stop             # 0-based exclusive
> > +        variant.end = record.start + len(record.ref)
>
> Ah, that makes sense. I was following the GA4GH documentation ("This is
> typically calculated by start + referenceBases.length") and wondering why
> my tests were failing. Thanks, I'll fix this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga4gh/server/pull/211/files#r25962648>.
>